ui/src/app/components/side-nav.spec.tsx

```diff
@@ -25,8 +16,13 @@ describe('SideNav', () => {
   });
 
   it('disables options when user not registered', () => {
-    const wrapper = mount(<SideNav {...props} profile={{...ProfileStubVariables.PROFILE_STUB, accessTierShortNames: []}}/>);
-    wrapper.setState({showUserOptions: true});
+    const wrapper = mount(<SideNav {...props} profile={{
+      ...ProfileStubVariables.PROFILE_STUB,
+      accessTierShortNames: [],
+      givenName: "Tester",
```

Comment:
we have all these xss injection testy things the actual profile stub for some godforsaken reason. this is the easiest way to actually click on the correct menu item.